index.js

```diff
@@ -16,7 +16,7 @@ module.exports = function dateHelper(str, pattern, options) {
   var opts = Object.assign({}, this, defaults, options, options.hash);
 
   // set the language to use
-  moment.locale(opts.lang);
+  moment.locale(opts.language);
```

Comment:
Why would you change this here? `opts.language` is a property that you set on your own context, correct?